mathics/core/element.py

```diff
@@ -106,18 +106,6 @@ class BaseElement(KeyComparable):
     # this variable holds a function defined in mathics.core.expression that creates an expression
     create_expression: Any
 
-    # FIXME: kwargs seems to be is needed because Real.__new_() takes a parameter
-    # and magically that gets turned into kwargs here.
-    # Figure out how to address this.
-    def __init__(self, *args, **kwargs):
-        self.options = None
-        self.pattern_sequence = False
-        # This property would be useful for a BoxExpression
-        # (see comment in mathocs.core.expression.) However,
-        # WL has a way to handle the connection between
-        # an expression and a Box expression ``InterpretationBox``.
-        # self.unformatted = self  # This may be a garbage-collection nightmare.
-
     # comment @mmatera: The next method have a name that starts with ``apply``.
     # This obstaculizes to define ``InstanceableBuiltin``
```

Comment:
If this commit goes in this comment is no longer valid and it would be nice to remove it then.

Reply:
Do you mean the one about ``apply_rule``? It is about the next method.

Reply:
Ah - you are correct. I thought it applied to the one above not following. 

Reading it though, it has some spelling and grammar mistakes. But that can be dealt with some other time.